lib/widgets/audio_bottom_sheet/audio_recitation_state_notifier.dart

```diff
@@ -60,8 +60,37 @@ class AudioRecitationStateNotifier extends StateNotifier<AudioRecitationState> {
   final AudioApi _audioApi;
   final AudioRecitationHandler _audioHandler;
 
+  final List<MediaItem> localPlaylist = [];
+
   StreamSubscription<PlayerState>? playerStateSubscription;
 
+  Future<void> getNextAyahMedia() async {
+    final bool shouldGoToNextSurah =
+        state.ayahId + 1 > (surahNumberToTotalAyahMap[state.surahId] ?? 0);
+
+    final int nextSurahId =
+        shouldGoToNextSurah ? state.surahId + 1 : state.surahId;
+    final int nextAyahNumber = shouldGoToNextSurah ? 1 : state.ayahId + 1;
+
+    final response = await _audioApi.getAudioForSpecificReciterAndAyah(
+      reciterId: state.reciterId!,
```

Comment:
agak oot, ini gue agak lupa kenapa reciterId nya dibuat nullable, tapi sedikit saran mungkin ini bisa dibikin default value reciterId nya 1. Di bagian state nya, jadi gaperlu ada tanda serunya gini. Kalau misal belum sempet nitip kasih comment // TODO: change to int instead of int?

Reply:
sabi ndras ane ubah sekalian.. tp gue setuju hrsnya bisa kita kasih default value

Reply:
updateddd